### PR TITLE
Integrate janus package

### PR DESCRIPTION
**Pull request by lsoucasse**
This work links Proteus with Janus as a package. Main differences are that Socrates is now a separate submodule and that input spectral data are downloaded from the OSF.

The main Proteus script seems to run fine but it might be worth checking it again.

### CODE REVIEW

**Review by nichollsh — CHANGES_REQUESTED** [review]
Looks great except for my 1 comment on the FWL_DATA variable.
The model seems to run fine on my machine too.

File: PROTEUS.env [hunk 1]
```diff
@@ -48,7 +42,7 @@ fi
 # Vulcan
 export PYTHONPATH=$COUPLER_DIR/VULCAN:$PYTHONPATH
 
-# Mors
+# Forming Worlds data
 export FWL_DATA=$COUPLER_DIR/fwl_data/
```
[the remark this answers is not in the document]
Comment (lsoucasse) [reply PROTEUS.env]: My idea was that it was safer for basic users. I agree it gives more flexibility to set it separately (I also store my fwl_data folder in another place). New commit should fix it.

**Review by nichollsh — APPROVED** [review]

**Comment by nichollsh** [comment]
Closes #100, I assume

**Comment by lsoucasse** [comment]
> Closes #100, I assume?

Thanks @nichollsh! Yes, the issue is linked with the branch and PR.

**Pull request merged**